.CI/update_all_tokens

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Expected to be run in a TravisCI context.
+
+set -e
+
+if [ -n "$GH_TOKEN" ]; then
+    # Install Miniconda.
+    echo ""
+    echo "Installing a fresh version of Miniconda."
+    curl -L https://repo.continuum.io/miniconda/Miniconda2-latest-MacOSX-x86_64.sh > ~/miniconda.sh
+    bash ~/miniconda.sh -b -p ~/miniconda
+    source ~/miniconda/bin/activate root
+
+    # Configure conda.
+    echo ""
+    echo "Configuring conda."
+    conda config --set show_channel_urls true
+    conda config --add channels conda-forge
+    conda install --yes --quiet git=2.12.2
```

Comment:
We should re-evaluate this pinning at some point, but it doesn't need to happen in this PR.

Reply:
I think I copied this script from a repo where there was an issue with the new git. Can't remember which one

Reply:
Yep, I think the pinning is my doing actually. Though I think it is from over a year ago and is probably not needed any more.

Reply:
Nope. it was me, https://github.com/conda-forge/staged-recipes/commit/3c7a57397ff764d71b285b97aba951c2d8db7027